commands/contributors.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
             .setFooter({ text: 'Please support the project using the following URL: https://github.com/jascaldeira/AOE4-DISCORD-BOT', iconURL: 'https://i.imgur.com/AfFp7pu.png' });
         embedData.setTitle('Contributors');
         embedData.addFields(
-            { name: '- ', value: '- Caldeira\n- PM303' }
+            { name: '- ', value: '- Caldeira\n- PM303\n- donPabloM' }
```

Comment:
just if you want, you can add yourself in the Readme file as well.

Reply:
nice, thanks